### PR TITLE
chore: add vale linting

### PR DESCRIPTION
**Pull request by jeroenleenarts**
## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [ ] Documentation is updated
- [ ] New code is tested in main example apps, including all possible scenarios
  - [ ] SampleApp iOS and Android
  - [ ] Expo iOS and Android

### CODE REVIEW

**Comment by madsroskar** [comment]
image

@jeroenleenarts at the moment, the linter is spell checking a whole bunch of propnames - would we have to add exceptions for each of these for this not to be an issue, or do you know of a less painful way of dealing with this?

File: docusaurus/docs/reactnative/basics/getting_started.mdx [hunk 1]
```diff
@@ -197,7 +197,7 @@ npm install react-native-video
 </TabItem>
 </Tabs>
 
-We require [exoplayer](https://github.com/google/ExoPlayer) to be the media player on Android. To make `react-native-video` to use exoplayer, add the following in `android/settings.gradle`:
+We require [ExoPlayer](https://github.com/google/ExoPlayer) to be the media player on Android. To make `react-native-video` to use ExoPlayer, add the following in `android/settings.gradle`:
```
Comment (madsroskar) [inline docusaurus/docs/reactnative/basics/getting_started.mdx:200]:
```suggestion
We require [ExoPlayer](https://github.com/google/ExoPlayer) to be the media player on Android. To make `react-native-video` use ExoPlayer, add the following in `android/settings.gradle`:
```

File: docusaurus/docs/reactnative/basics/hello_stream_chat.mdx [hunk 2]
```diff
@@ -233,7 +233,7 @@ export const App = () => (
 Stream Chat for React Native provides a `ChannelList` component out of the box for displaying a list of channels.
 `ChannelList` can be used with no props and will return all of the channels the set user has access to.
 To functionally use the `ChannelList` an `onSelect` prop should be provided.
-The `onSelect` function is called with the selected `Channel` instance and can then be used to react to the user interaction, e.g. store the `Channel` in state and navigate to the appropriate channel screen.
+The `onSelect` function is called with the selected `Channel` instance and can then be used to react to the user interaction, for example store the `Channel` in state and navigate to the appropriate channel screen.
```
Comment (madsroskar) [inline docusaurus/docs/reactnative/basics/hello_stream_chat.mdx:236]:
```suggestion
The `onSelect` function is called with the selected `Channel` instance and can then be used to react to the user interaction, for example to store the `Channel` in state and navigate to the appropriate channel screen.
```

File: docusaurus/docs/reactnative/basics/troubleshooting.mdx [hunk 3]
```diff
@@ -53,7 +53,7 @@ Ensure the `keyboardVerticalOffset` prop passed to `Channel` is correct and acco
 
 ### Android
 
-On Android if the StatusBar is set to translucent, i.e. `StatusBar.setTranslucent(true)` then there may be some inaccurate layout calculations occurring.
+On Android if the StatusBar is set to translucent, that is `StatusBar.setTranslucent(true)` then there may be some inaccurate layout calculations occurring.
```
Comment (madsroskar) [inline docusaurus/docs/reactnative/basics/troubleshooting.mdx:56]:
```suggestion
On Android if the StatusBar is set to translucent, that is `StatusBar.setTranslucent(true)`, there may be some inaccurate layout calculations occurring.
```

File: docusaurus/docs/reactnative/basics/troubleshooting.mdx [hunk 4]
```diff
@@ -108,7 +108,7 @@ Both `bottomInset` & `topInset` can be set via the appropriate functions, `setBo
 
 ### Android
 
-On Android if the StatusBar is set to translucent, i.e. `StatusBar.setTranslucent(true)` then you need to provide the prop `translucentStatusBar` to the `OverlayProvider`.
+On Android if the StatusBar is set to translucent, that is `StatusBar.setTranslucent(true)` then you need to provide the prop `translucentStatusBar` to the `OverlayProvider`.
```
Comment (madsroskar) [inline docusaurus/docs/reactnative/basics/troubleshooting.mdx:111]:
```suggestion
On Android if the StatusBar is set to translucent, that is `StatusBar.setTranslucent(true)`, you need to provide the prop `translucentStatusBar` to the `OverlayProvider`.
```

File: docusaurus/docs/reactnative/customization/typescript.mdx [hunk 5]
```diff
@@ -89,7 +89,7 @@ type DefaultUserType = Record<string, unknown> & {
 };
 ```
 
-Additional fields on the defaults, i.e. `image`, are custom fields used by `stream-chat-react-native` already within the SDK.
+Additional fields on the defaults, that is `image`, are custom fields used by `stream-chat-react-native` already within the SDK.
```
Comment (madsroskar) [inline docusaurus/docs/reactnative/customization/typescript.mdx:92]:
```suggestion
Additional fields on the defaults, for example `image`, are custom fields used by `stream-chat-react-native` already within the SDK.
```

I think i.e. was used incorrectly here

**Review by madsroskar — APPROVED** [review]
From scrolling through and skimming the changes (too many to go into very much detail), just added some suggestions where things sounded a bit clunky. 

Looks good to me though :)

**Comment by DaemonLoki** [comment]
Good catches @madsroskar! That's one limitation of `vale`. It doesn't really do grammar(ly)-style checks but it catches quite a bit of other ones.

But I think it's always worth doing the scout rule and when touching something and encountering errors just fix those, as you did. :) 

So, this seems to be ready to be merged, right?

**Pull request merged**